### ISSUE TITLE
Prefix fails with second selector in multiple selectors

### ISSUE THREAD

**Issue by Kreozot** [justim/postcss-svg-fallback#13]
For example:

```
.my_class_1, .my_class_2 {...}
```

Result:

```
.no-svg .my_class_1, .my_class_2 {...}
```

So the second selector doesn't have no-svg class in it.

**Comment by Kreozot** [comment]
Added test for this issue:
https://github.com/Kreozot/postcss-svg-fallback/commit/756ad640b46d7137ffdd8269daa917de538d9d66

**Comment by Kreozot** [comment]
Fixed.

**Comment by justim** [comment]
Published as version 1.2.2 :+1:
